test/joke_test.rb

```diff
@@ -14,4 +14,9 @@ def test_it_has_attributes
     assert_equal "Why did the strawberry cross the road?", joke.setup
     assert_equal "Because his mother was in a jam.", joke.punchline
   end
+
+  def test_joke_2_is_added
```

Comment:
Since we have verified that a joke can be created in the first test, we don't need to verify it again.